packages/oid4vci-demo-frontend/src/configs/legal-entity.banqup.json

```diff
@@ -189,6 +192,30 @@
           "height": 65
         }
       }
+    },
+    "issueSuccess": {
+      "leftPaneWidth": "40%",
+      "backgroundColor": "#0E0B31",
+      "logo": {
+        "src": "banqup/logo.svg"
+      },
+      "rightPaneTitle": "credentials_success_right_pane_valid_id_title",
+      "rightPaneParagraph": "credentials_success_right_pane_registered_company_paragraph",
+      "rightPaneButtonCaption": "label_finish",
+      "rightPaneTextHeight": "40%",
+      "rightPaneTextMarginTop": "0",
+      "rightPaneTextMarginBottom": "10%",
+      "rightPaneButtonWidth": "135px",
```

Comment:
if it is a small fix, can we start using numbers only for widths and heights? i mentioned this on another pr as well. i think we use these only for style props. and react handles the units for you